master/security/host-forwarded-traffic.md

```diff
@@ -0,0 +1,158 @@
+---
+
+### Big picture
+
+Enforce {{site.prodname}} policy on traffic transiting a host that is used as a router or NAT gateway.
+
+### Value
+
+If your host has multiple network interfaces, and is configured as a router, or NAT gateway between two different networks, you may want to enforce policy on traffic as it moves between the networks. In this configuration, often neither the source or destination is a {{site.prodname}} endpoint, so policy enforcement at the endpoint is not available. You can centrally manage the firewall policy on a fleet of such hosts using the same policy language as the rest of {{site.prodname}}.
+
+### Features
+
+This how-to guide uses the following {{site.prodname}} features:
+
+- **Host Endpoint**
+- **GlobalNetworkPolicy**
+  - applyOnForward setting
+  - preDNAT setting
+
+### Concepts
+
+#### Workload endpoints and host endpoints
+
+The following figure shows a host with two network interfaces: eth0 and eth1. We call these **host endpoints (HEPs)**. The host also runs two guest workloads (VMs or containers). We call the virtual interfaces to the guests, **workload endpoints (WEPs)**.  Each has a corresponding configuration object on the {{site.prodname}} API called HostEndpoint and WorkloadEndpoint, respectively. 
+
+The `HostEndpoint API` object is optional, and Calico does not enforce any policy on the HEP if the API object is missing. The `WorkloadEndpoint API` object is required, and is automatically managed by the cluster orchestrator plugin (for example, Kubernetes or OpenStack).
+
+Several connections are shown in the figure, numbered 1 through 4. For example, connection 1 ingresses over HEP eth0, is forwarded, and then ingresses Workload A’s WEP. Calico policies select which WEPs or HEPs they apply to. So, for example an ingress policy that selects Workload A’s WEP will apply to connections as shown in number 1.
+
+![Host-forward-traffic]({{site.baseurl}}/images/host-forward-traffic.png)
+
+#### applyOnForward
+
+By default, {{site.prodname}} global network policies set **applyOnForward to false**. When set to false on policies that select HEPs, the policies are applied only to traffic that originates or terminates on the host, for example: connection 4 (Node process). Connections 1-3 are unaffected by policies that select the HEP, but have applyOnForward set to false.
+
+In contrast, if applyOnForward is set to true for a policy that selects a HEP, that policy can apply to all connections 1-4. For example:
+
+- Ingress policy on HEP eth0 affects connections 1 and 2
+- Egress policy on HEP eth1 affects connections 2, 3, and 4
+
+There are also different default action semantics for forwarded traffic (connections 1-4) versus local traffic (connection 4) when it traverses a HEP.  If no applyOnForward policy selects the HEP and direction (ingress versus egress), then forwarded traffic is allowed.  If no policy (regardless of applyOnForward) selects the HEP and direction, then local traffic is denied.
```

Comment:
> forwarded traffic (connections 1-4) versus local traffic (connection 4) 

Isn't the forwarded traffic just connections 1-3?

Reply:
Yes, that sentence is not clear. Better?